site/content/docs/developing/observability.ja.md

```diff
@@ -0,0 +1,119 @@
+# 可観測性 (Observability)
+
+## はじめに
+Copilot では、Manifest にて以下のように設定することにより、Service のトレース収集を設定することができます。
+```yaml
+observability:
+  tracing: awsxray
+```
+
+[Request-Driven Web Service](../concepts/services.ja.md#request-driven-web-service) の場合、Copilot は App Runner に組み込まれた[トレース設定](https://docs.aws.amazon.com/ja_jp/apprunner/latest/dg/monitor-xray.html)を有効にします。
+
+[Load-Balanced Web Service](../concepts/services.ja.md#load-balanced-web-service)、[Backend Service](../concepts/services.ja.md#backend-service)、[Worker Service](../concepts/services.ja.md#worker-service) の場合、Copilot は AWS OpenTelemetry Collector を[サイドカー](./sidecars.ja.md)としてデプロイします。
```

Comment:
Could you remove '-' between 'Load' and 'Balanced' so that the first service is 'Load Balanced Web Service'?

Reply:
okay